frontend/packages/knative-plugin/src/components/functions/GettingStartedSection.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Card, CardBody, ExpandableSection } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { FunctionsDocsGettingStartedCard } from './FunctionsDocsGettingStartedCard';
+import { QuickStartGettingStartedCard } from './QuickStartGettingStartedCard';
+import { SampleGettingStartedCard } from './SamplesGettingStartedCard';
+
+import './GettingStartedSection.scss';
+
+export const GettingStartedSection: React.FC = () => {
+  const { t } = useTranslation();
+  const [isOpen, setIsOpen] = React.useState(true);
```

Comment:
Can we save that state in the user settings (defaultValue=true, sync=false)?

So that the box is closed when I open the functions list page later again.